src/lib/registration/helpers.rb

```diff
@@ -85,11 +85,18 @@ def self.service_description(service)
       (descr && !descr.empty?) ? "#{descr} (#{url})" : url
     end
 
+    # run the network configuration module
     def self.run_network_configuration
       log.info "Running network configuration..."
       Yast::WFM.call("inst_lan", [{ "skip_detection" => true }])
     end
 
+    # check if the network configuration module is present
+    # @return [Boolean] true if the network configuration client is present
+    def self.network_configurable
+      Yast::WFM.ClientExists("inst_lan")
```

Comment:
Is this false for installed systems? I have inst_lan.rb in my system

Or is simply safe to call that client in an already installed system?